mavros/src/plugins/setpoint_position.cpp

```diff
@@ -180,10 +185,45 @@ class SetpointPositionPlugin : public plugin::PluginBase,
 		send_position_target(req->header.stamp, tr);
 	}
 
+	/**
+	 * Gets setpoint position setpoint and send SET_POSITION_TARGET_GLOBAL_INT
+	 */
+	void setpointg_cb(const geographic_msgs::GeoPoseStamped::ConstPtr &req)
+	{
+		Eigen::Vector3d velocity, af;
+		Eigen::Quaterniond attitude;
+		float yaw, yaw_rate;
+		uint16_t type_mask;
+
+		uint16_t type_mask = mavros_msgs::GlobalPositionTarget::IGNORE_VX 
```

Comment:
do not use `GlobalPositionTarget` enum here. Use the same approach for setting the mask as in `send_position_target()` method.